helpers/deploymentConfig.ts

```diff
@@ -3000,3 +3003,20 @@ export async function getTokenAddress(tokenConfig: TokenConfig, deployments: Dep
     return tokenConfig.tokenAddress;
   }
 }
+
+export const getVWNativeToken = (networkName: string) => {
+  switch (networkName) {
+    case "hardhat":
+      return ilBscTestnet.VToken_vWBNB_LiquidStakedBNB.address;
+    case "bsctestnet":
+      return ilBscTestnet.VToken_vWBNB_LiquidStakedBNB.address;
+    case "bscmainnet":
+      return ilBscMainnet.VToken_vWBNB_LiquidStakedBNB.address;
+    case "ethereum":
+      return ilEthereum.VToken_vWETH_Core.address;
+    case "development":
+      return ilBscTestnet.VToken_vWBNB_LiquidStakedBNB.address;
```

Comment:
I think we don't use the network `development`. On the other hand, I miss the `sepolia` entry

Reply:
Done https://github.com/VenusProtocol/isolated-pools/pull/361/commits/b508cb1ea7a7fd40bba305ac47aca94f903aae9f